.github/CODEOWNERS

```diff
@@ -61,10 +61,10 @@ sycl/doc/extensions/experimental/sycl_ext_intel_esimd/ @intel/dpcpp-esimd-review
 llvm/lib/SYCLLowerIR/CMakeLists.txt @intel/dpcpp-tools-reviewers @intel/dpcpp-esimd-reviewers
 
 # invoke_simd
-sycl/include/sycl/ext/oneapi/experimental/invoke_simd.hpp @rolandschulz @kbobrovs
-sycl/include/std/experimental/simd.hpp @rolandschulz @kbobrovs
-llvm/lib/SYCLLowerIR/LowerInvokeSimd.cpp @kbobrovs @v-klochkov @rolandschulz
-llvm/include/llvm/SYCLLowerIR/LowerInvokeSimd.h @kbobrovs @v-klochkov @rolandschulz
+sycl/include/sycl/ext/oneapi/experimental/invoke_simd.hpp @intel/dpcpp-esimd-reviewers
+sycl/include/std/experimental/simd.hpp @intel/dpcpp-esimd-reviewers
+llvm/lib/SYCLLowerIR/LowerInvokeSimd.cpp @intel/dpcpp-esimd-reviewers
+llvm/include/llvm/SYCLLowerIR/LowerInvokeSimd.h @intel/dpcpp-esimd-reviewers
```

Comment:
Thanks for making this change. I was going to replace kbobrovs with intel/dpcpp-esimd-reviewers and already discussed this change with @v-klochkov.
@rolandschulz, do you still want to be added to reviewers?

Reply:
Happy to still review the parts in sycl/include. Don't consider myself a good reviewer for the llvm/ parts.